plugins/techdocs/src/client.ts

```diff
@@ -124,18 +120,15 @@ export class TechDocsClient implements TechDocsApi {
 export class TechDocsStorageClient implements TechDocsStorageApi {
   public configApi: Config;
   public discoveryApi: DiscoveryApi;
-  public identityApi: IdentityApi;
   private fetchApi: FetchApi;
 
   constructor(options: {
     configApi: Config;
     discoveryApi: DiscoveryApi;
-    identityApi: IdentityApi;
```

Comment:
This would be a breaking change as is, do we want to make this backwards compatible?

Reply:
I'm not aware of the implications, I just removed the reference as it is not used explicitly any more.

Is there an option for a soft deprecation? shouldn't plugin initialization already be hidden by the new backend system in the alpha.ts, making this breaking change less impactful?

Reply:
Up to the maintainers here, but I think we could just keep it on the constructor and mark it as optional. That may not be worth the effort though, I'm not sure how strict we generally are with API changes.